src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -937,35 +948,33 @@ static int dhcp_server_relay_message(sd_dhcp_server *server, DHCPMessage *messag
         return -EBADMSG;
 }
 
-static int prepare_new_lease(
-                DHCPLease **ret_lease,
-                be32_t address,
-                const DHCPClientId *client_id,
-                uint8_t htype,
-                uint8_t hlen,
-                const uint8_t *chaddr,
-                be32_t gateway,
-                usec_t expiration) {
+static int prepare_new_lease(DHCPLease **ret_lease, be32_t address, DHCPRequest *req, usec_t expiration) {
 
         _cleanup_(dhcp_lease_freep) DHCPLease *lease = NULL;
 
         lease = new(DHCPLease, 1);
         if (!lease)
                 return -ENOMEM;
 
-        *lease = (DHCPLease) {
+        *lease = (DHCPLease){
```

Comment:
Please do not drop the space before `{`.

Reply:
Fixed. Seems like clang-format removed it for some reason...